### ISSUE TITLE
"Resistance is futile" - bad association for backups

### ISSUE THREAD

**Issue by TinajaLabs** [borgbackup/borg#4195]
"Resistance is futile" --the borg

Not sure if you get it that the borg has a very negative connotation.  It's a metaphor for societal conformance to the blind takeover of everything; like Amazon.com, Facebook, or Trump's denial of science, climate change, and the labeling of news organizations as "enemy of the people".  It feels like I will be sharing my valuable assets with the collectively shared backups of all those who use "the borg" whether I like it or not.

Not a good association.

How about something less threatening like "Secure Dedupie"...  ;)

I do appreciate your efforts.

Thanks,
Chris.

**Comment by ThomasWaldmann** [comment]
not sure about blind takeover of everything. don't the borg choose and only assimilate the good stuff?

some points about "why 'borg'":
- deduplication (and also duplication, at restore time) is related to *clones*
- all your data will end up in (your) borg repo collective
- the borg (developer) collective will assimilate advanced technology if they see it fit
- you can make startrek jokes and print them in ridiculously big fluorescent green letters on black borgbackup t-shirts

some people upload their borg repos to the amazon cloud in trump country (not with borg, but otherwise) - so that's maybe getting close to what you posted, haha. but, it's still encrypted and authenticated, so not as bad as it sounds, assuming they can download it successfully when needed.

http://www.linux-magazine.com/Issues/2018/210/BorgBackup

**Comment by ThomasWaldmann** [comment]
https://en.wikipedia.org/wiki/Borg_(castle) you can alternatively associate with that kind of `borg`, if you prefer.

**Comment by danieldjewell** [comment]
FWIW, as a Star Trek fan myself, @ThomasWaldmann makes a great point - "The Borg" only assimilate things that will *add* to their own _perfection_ -- as I recall, there were several canon instances where they refused to assimilate planets/races/species because they considered them "lesser" and would not add to their _perfection_. 

That said, the word itself has [quite a few meanings](http://en.wiktionary.org/wiki/borg) -- including "surety/guarantor/deposit" in Dutch. Not to mention that it's the name of places, and various other things - including a relatively common surname in Malta [Wikipedia - Borg (disambiguation)](https://en.wikipedia.org/wiki/Borg_(disambiguation)). Not to mention the Swedish Vodka brand. (If I recall my German correctly, I believe "die Burg" would be the cognate with "borg" in the various other Germanic languages like Danish, Swedish, etc. as @ThomasWaldmann pointed out. Perhaps renaming the project to "Burg" would work - sadly, that would remove so much of the fun!) 

That said, IMHO, @TinajaLabs I think that while some may indeed associate the word "borg" with the alien race depicted in _Star Trek_, that would be their view and that would be their limitation on seeing the word in a limited way based on their cultural experiences. While I fully recognize that certain words do indeed have a negative meaning and carry with them a negative connotation, I also cannot help but think that if someone has such a strong rejection based on the meaning of the word (which in my view, seems to be used in a playful and fun way), perhaps they should choose to not use the free and open-source software as is their right in an open and free society. 

In other words: optimally, everyone should take things in stride, laugh, and enjoy. I acknowledge that not everyone will do that and nothing I say can change that. But that would be the optimal situation. 

There's an expression in German that encompasses what I'm trying to say -- it's not really "Gemütlichkeit" but perhaps "Du solltest lockerer werden". (Best idiomatic translation I can think of -- "Roll with it".) 

I'd say - keep the name. It's cool. If people have a problem with it - well.... okay.

**Comment by fantasya-pbem** [comment]
Never judge something by its name. Borg or BorgBackup is a cool name.

**Comment by gagan0123** [comment]
I use it in conjunction with rsync on my Raspberry Pi.
Rsync first syncs the data from servers to local drive, then borg is used for incremental backups.
I do this because every time there's a need to restore a backup, rsync backups are the only ones that worked, after using a lot of other backup softwares.

One thing that brought me to this backup utility is the name "Borg" I like the borg collective and am a star trek fan.

~Just another resisting rsync user 😉

**Comment by ThomasWaldmann** [comment]
As there was the question in my inbox whether I'ld consider renaming the project:

No. :-)

I like the name due to the reasons stated above and everybody who does not like it (in the startrek sense) has non-startrek options, too, see above.

**Comment by ThomasWaldmann** [comment]
<img width="1408" height="768" alt="Image" src="https://github.com/user-attachments/assets/668f59a9-2d5e-4849-983b-432e14e69bd8" />

**Comment by ThomasWaldmann** [comment]
(that's made with nano banana)

**Comment by gagan0123** [comment]
Just to add, if you follow the series, the perception of Borg changed quite a lot in the Star Trek: Picard series. They went from being perceived as threats to being victims and ultimately heroic allies who sacrificed to save federation.

**Comment by ThomasWaldmann** [comment]
Fable5 was quite creative last night (short before they killed it for most people):
- Functionality: "Redundancy is futile. Your data will be deduplicated."
- Support: "You are not alone. It's a collective."
- Funding: "Resistance is futile. Donations are not."

**Comment by ThomasWaldmann** [comment]
Guess we can all agree with "Redundancy is futile. Your data will be deduplicated." (at least when looking at a single backup location, knowing that one should have multiple backups at different locations).
